packages/docs-app/src/examples/core-examples/multistepDialogExample.tsx

```diff
@@ -115,7 +115,10 @@ export class MultistepDialogExample extends React.PureComponent<
                     icon="info-sign"
                     navigationPosition={position}
                     onClose={this.handleClose}
-                    nextButtonProps={{ disabled: this.state.value === undefined }}
+                    nextButtonProps={{
+                        disabled: this.state.value === undefined,
+                        tooltipContent: this.state.value === undefined ? "Needs an input." : null,
```

Comment:
no need to use `null` here:

```suggestion
                        tooltipContent: this.state.value === undefined ? "Needs an input" : undefined,
```